packages/api-electron/index.js

```diff
@@ -10,16 +10,48 @@ const {
   IPC_SSF_BLUR_WINDOW,
   IPC_SSF_CAPTURE_SCREEN_SNIPPET,
   IPC_SSF_CLOSE_WINDOW,
+  IPC_SSF_FLASH_FRAME,
   IPC_SSF_FOCUS_WINDOW,
+  IPC_SSF_GET_WINDOW_BOUNDS,
   IPC_SSF_GET_WINDOW_ID,
+  IPC_SSF_GET_WINDOW_MAXIMUM_SIZE,
+  IPC_SSF_GET_WINDOW_MINIMUM_SIZE,
+  IPC_SSF_GET_WINDOW_PARENT,
+  IPC_SSF_GET_WINDOW_POSITION,
+  IPC_SSF_GET_WINDOW_SIZE,
+  IPC_SSF_GET_WINDOW_TITLE,
   IPC_SSF_HIDE_WINDOW,
+  IPC_SSF_IS_WINDOW_ALWAYS_ON_TOP,
+  IPC_SSF_IS_WINDOW_MAXIMIZABLE,
+  IPC_SSF_IS_WINDOW_MAXIMIZED,
+  IPC_SSF_IS_WINDOW_MINIMIZABLE,
+  IPC_SSF_IS_WINDOW_MINIMIZED,
+  IPC_SSF_IS_WINDOW_RESIZABLE,
+  IPC_SSF_LOAD_URL,
+  IPC_SSF_MAXIMIZE_WINDOW,
+  IPC_SSF_MINIMIZE_WINDOW,
   IPC_SSF_NEW_WINDOW,
   IPC_SSF_NOTIFICATION,
+  IPC_SSF_RELOAD_WINDOW,
+  IPC_SSF_RESTORE_WINDOW,
   IPC_SSF_SCREEN_SNIPPET_CAPTURED,
+  IPC_SSF_SET_WINDOW_ALWAYS_ON_TOP,
+  IPC_SSF_SET_WINDOW_BOUNDS,
+  IPC_SSF_SET_WINDOW_ICON,
+  IPC_SSF_SET_WINDOW_MAXIMIZABLE,
+  IPC_SSF_SET_WINDOW_MAXIMUM_SIZE,
+  IPC_SSF_SET_WINDOW_MINIMIZABLE,
+  IPC_SSF_SET_WINDOW_MINIMUM_SIZE,
+  IPC_SSF_SET_WINDOW_POSITION,
+  IPC_SSF_SET_WINDOW_RESIZABLE,
+  IPC_SSF_SET_WINDOW_SIZE,
+  IPC_SSF_SET_WINDOW_SKIP_TASKBAR,
   IPC_SSF_SEND_MESSAGE,
   IPC_SSF_SHOW_WINDOW,
   IPC_SSF_WINDOW_EVENT,
-  IPC_SSF_WINDOW_SUBSCRIBE_EVENTS
+  IPC_SSF_WINDOW_HAS_SHADOW,
+  IPC_SSF_WINDOW_SUBSCRIBE_EVENTS,
+  IPC_SSF_UNMAXIMIZE_WINDOW
```

Comment:
What do you think about having a general `const { constants } = require('constants')` as well as the individual ones so we can import all of these in one go (as there's so many)?